tbtaf/tbtaf_launcher.py

```diff
@@ -17,15 +18,16 @@
             tbtafcli.execute()
             
     else:
-        testScript = './test/test02.tbtaf'
-        myTBTAF = TBTAFOrchestrator()
+        testScript = './test/test05.tbtaf'
+        targetDatabridge = TBTAFSqliteDatabridge()
```

Comment:
Let's have this configurable, for now its ok that this will be the default, but lets define some argument to define the dataBridgeType and then initialize based on that